<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NODTBSL175AG" name="Firefox Result" tests="2" failures="2" timestamp="3 Jul 2015 05:28:16 GMT" time="26.03" errors="0">
  <testcase name="open_contact_us" time="1.713" classname="testcase.Unregistered_User">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
	at testcase.Unregistered_User.open_contact_us(Unregistered_User.java:25)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- open_contact_us -->
  <testcase name="open_contact_us" time="1.718" classname="testcase.Unregistered_User">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="0">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 0
	at util.OnFailure.onTestFailure(OnFailure.java:43)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- open_contact_us -->
</testsuite> <!-- Firefox Result -->
